examples/standalone-many-tables/data-policy.yaml

```diff
@@ -2,9 +2,6 @@ metadata:
   description: ""
   version: 1
   title: public.demo
-platform:
-  id: standalone-sample-connection
```

Comment:
The platform has been removed, and is now part of `ResourceUrn`. However, I don't see any mention of the platform in these example policies anymore. How is that handled now?